examples/MyPackage/ui/ipc/MyAppIPCServer.qml

```diff
@@ -51,5 +50,10 @@ Window {
         text: myInterface.counter
     }
 
+    MouseArea {
```

Comment:
Can you apply the changes (if needed at all) to MyAppInProcess.qml, as well (should be the same code and show in-process behaviour)?